### DISCUSSION TITLE
No activation email received for forum sign up

### DISCUSSION THREAD

**Discussion by stephent** [strapi/strapi#8838] [Legacy Topics]
I'm sorry to see that you'll be closing Discussions here - it's a more convenient place for developers than separate forums, I think.

However, one issue is that I cannot sign up for the forum because no activation email is ever received (four requests so far over several days). The email address I registered is correct and is receiving emails from other senders just fine. (And no, it's not in spam either.)

Hope you can help resolve before you shut this channel down 😁 ?

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
